src/base1/cockpit.js

```diff
@@ -1468,19 +1468,20 @@ function factory() {
          * non-localised conversions (and in both cases, show no
          * fractional part).
          */
+	var lang = cockpit.language === undefined ? undefined : cockpit.language.replace('_', '-');
```

Comment:
Can it happen that it's `null`? Maybe just check for truthness? `cockpit.language ? …replace() : 'en'`? (Or default to null if the code below is able to handle that).

Reply:
The value is initialised to 'undefined', which is the case I'm checking for here.  Any call to locale() will be with a po file, so I don't think we will ever see null here...

I'll change it if you think it would be better.

Reply:
Thanks, that's fine. Just wanted to cover all bases.